openmls/tests/book_code.rs

```diff
@@ -828,9 +828,19 @@ fn book_operations(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvide
         unreachable!("Expected a QueuedProposal.");
     }
 
+    // Create AddProposal and remove it
+    // ANCHOR: remove_proposal_by_ref
```

Comment:
Just a nit, but `remove_proposal_by_ref` sounds confusing since we have an add proposal. Maybe 'roll back'?

Reply:
Would it work if I put the add proposal out of the anchor? Or you prefer to remove the doc completely regarding this?

Reply:
I've just got your comment. You meant to rename the anchor, right?